modules/logging-loki-logcli.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+// 
+//
+:_content-type: REFERENCE
+[id="logging-loki-logcli-about_{context}"]
+
+= Querying Loki
+
+You can use Loki's command-line interface `logcli` to query logs.
+
+.Example Application Log Query
+[source,terminal]
+----
+$ oc extract cm/lokistack-sample-ca-bundle --to=lokistack --confirm
+$ cat lokistack/*.crt >lokistack_ca.crt
+$ logcli -o raw --bearer-token="${bearer_token}" --ca-cert="lokistack_ca.crt" --addr xxxxxx
+----
+
+.Example Infrastructure Log Query
+[source,terminal]
+----
+$ logcli --bearer-token="$(oc whoami -t)" --addr https://lokistack-dev-openshift-logging.apps.devcluster.openshift.com/api/logs/v1/infrastructure labels
+----
+
+.Example Audit log Query
+[source,terminal]
+----
+$ logcli --bearer-token="$(oc whoami -t)" --addr https://lokistack-dev-openshift-logging.apps.devcluster.openshift.com/api/logs/v1/audit labels
+----
+
+.Additional Resources
```

Comment:
See guidance for ID for additional resources: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections